Header header
geometry_msgs/PointStamped[] positions
